14/umbraco-forms/tutorials/creating-a-multipage-form.md

```diff
@@ -0,0 +1,161 @@
+7. Click **Submit**.
+8. Click **Save**.
+
+![Company information](images/company-information.png)
+
+Similarly, you can also add other relevant fields based on your requirements.
+
+### Add Conditional Logic
+
+Umbraco Forms allows you to customize the flow of your multi-page form. You can add conditional logic to control which questions appear based on user inputs.
+
+To add conditions, follow these steps:
+
+1. Click **Add question**.
+2. Select **Single choice** as the field type.
+3. **Enter a Name** for the field type. For example, *Do you work with Umbraco*.
+4. Enter the **Value** and **Caption** in the **Options** field.
+
+![Options Field](images/prevalue-fields.png)
+
+5. Mark the field as **Mandatory**.
+6. Click **Submit**.
+7. Click **Save**.
+8. Repeat steps 1-4 to create a conditional question titled *If yes, how many years?*
+
+![Next Question](images/conditional-question-part-1.png)
+
+9. Enable **Conditions**.
+10. Set the parameters for the condition as follows:
+
+![Add condition](images/conditional-question-part-2.png)
+
+11. Click **Submit**.
+12. Click **Save**.
+
+![Second Page Details](images/page-2-details.png)
+
+## Create the Final Page
+
+1. Click **Add new page** to create the final page of your form.
+2. **Enter a Name** for this page. For example: *Products*.
+3. Click **Add question**.
+4. Select **Multiple choice** as the field type.
+5. **Enter a Name** for the field type. For example, *Select the products you are interested in*.
+6. Enter the **Value** and **Caption** in the **Options** field.
+
+![Question Options](images/multiple-choice.png)
+
+7. Click **Submit**.
+8. Click **Save**.
+
+![Final Page Details](images/Final-page.png)
+
+## Embed the Form on a Web Page
+
+Once you are satisfied with your multi-page form, it is time to embed it on your website.
+
+To display the form on the website, follow these steps:
+
+1. Navigate to the **Content** section of Umbraco.
+2. Click **+** next to the parent page of the website.
+3. **Enter a Name** for the page. For example, *Book a Meeting!*
+4. Select the **Book a Meeting** form using the *Form Picker*.
+5. Click **Save and Publish**.
+
+![Content Section](images/Form-Content-section.png)
+
+## Customize Form Settings
+
+If you wish to customize the Form Settings, see the [Form Settings](../editor/creating-a-form/form-settings.md) article.
+
+## Rendering the Form on the Frontend
+
+To render the Form on the frontend, use the following code in your template:
+
+```cs
+@await Component.InvokeAsync("RenderForm", new { formId = Guid.Parse("<form guid>"), theme = "default", includeScripts = false })
+```
+
+For more information, see the [Rendering Forms](../developer/rendering-forms.md) article.
+
+{% hint style="info" %}
+For Umbraco Forms to work correctly, you need to include some client dependencies. For more information, see the [Preparing Your Frontend](https://docs.umbraco.com/umbraco-forms/developer/prepping-frontend) article.
+{% endhint %}
```

Comment:
A suggestion for this entire section:

Start by directing the user to the Preparing your frontend article.
Then direct the user to the Rendering Forms article.

That way this section is slimmed down, and we do not end up repeating content that is already written in those articles.

What do you think about that idea? 🤔 

Side note: The Rendering Forms article need to be updated as well, as it still mentions Macros 😅 I'll add a task for it!

Reply:
I like it. I've removed the text from the Note and placed it at the start.

Awesome. I noticed the Macro in the article but did not add a task to remove it 🙈